### PR TITLE
Odd occurences in array

### PR DESCRIPTION
**Pull request by wozane**

### CODE REVIEW

File: odd_occurrences/odd_occurrences.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def solution(array)
+  numbers = Hash.new(0)
```
Comment (esse) [inline odd_occurrences/odd_occurrences.rb]: is it really needed here?

File: odd_occurrences/odd_occurrences.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  odd_numbers = Hash.new(0)
```
Comment (esse) [inline odd_occurrences/odd_occurrences.rb]: same question as before

File: odd_occurrences/odd_occurrences.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+  numbers = array.group_by { |num| num }
```
Comment (esse) [inline odd_occurrences/odd_occurrences.rb]: you can write it prettier: `numbers = array.group_by(&:itself)`

**Review by esse — CHANGES_REQUESTED** [review]
style changes

File: odd_occurrences/odd_occurrences.rb (hunk 2 above)
Comment (wozane) [reply odd_occurrences/odd_occurrences.rb]: 👍

File: odd_occurrences/odd_occurrences.rb (hunk 1 above)
Comment (wozane) [reply odd_occurrences/odd_occurrences.rb]: not really :)

**Pull request merged**